### ISSUE TITLE
App displays different locales incorrectly

### ISSUE THREAD

**Issue by hissingpanda** [RefugeRestrooms/refugerestrooms-android#7]
![screenshot_2015-08-16-14-23-40](https://cloud.githubusercontent.com/assets/7660115/9294821/2c3b6a6c-441e-11e5-8185-8f2792f28819.png)

Displays weird symbols in place of the accents in french (and other languages I'm sure). Makes the app largely unusable in places like Montreal I imagine, so I'll try to roll out an update this week fixing that.

**Comment by hissingpanda** [comment]
Might just be related to the particular font, or the text file encoding

**Comment by hissingpanda** [comment]
Fixed this. Was an issue with the file encoding. I had to encode the "Name" and "Directions" results into ISO-8859-1 and then decode them back into UTF-8 in order for the characters to display correctly, which is pretty weird. Pushing the update to beta for a quick test and then live.

I think this problem might actually be a redundancy in the way the server API encodes. Similar to this PR: https://github.com/RefugeRestrooms/refugerestrooms/pull/60. 

Text displays correctly on the Web app (because of that PR) but incorrectly when going to the API link (e.g http://www.refugerestrooms.org/api/v1/restrooms/search.json?query=Grande%20Biblioth%C3%A8que%20du%20Qu%C3%A9bec). Which requires an encoding translation on the API user's part.
